impeller/renderer/backend/vulkan/swapchain_impl_vk.cc

```diff
@@ -343,10 +343,10 @@ SwapchainImplVK::AcquireResult SwapchainImplVK::AcquireNextDrawable() {
   /// Get the next image index.
   ///
   auto [acq_result, index] = context.GetDevice().acquireNextImageKHR(
-      *swapchain_,                           // swapchain
-      std::numeric_limits<uint64_t>::max(),  // timeout (nanoseconds)
-      *sync->render_ready,                   // signal semaphore
-      nullptr                                // fence
+      *swapchain_,          // swapchain
+      1'000'000'000,        // timeout (ns) 1000ms
```

Comment:
I was about to say, "if this is a magic number anyway, let's make it the same as the Metal backend.". The docs for `[CAMetalLayer nextDrawable]` say "If all drawables are in use, the layer waits up to one second for one to become available, after which it returns nil.". If this was intentional, amazing attention to detail!

Reply:
😉